docs/tutorials/hub/user-thread-database.md

```diff
@@ -0,0 +1,194 @@
+# Thread Database
+
+In this tutorial, we'll walk through using the local Thread Database in JavaScript. The `Database` can be used to store and work with data entirely locally in your app and it can be linked to other nodes that can help sync, rely, or store data when the user goes offline. We'll use those features to create a Database, connect it to the Hub, and connect two peers.
```

Comment:
```suggestion
In this tutorial, we'll walk through using the local Thread Database in JavaScript. The `Database` can be used to store and work with data entirely locally in your app and it can be linked to other nodes that can help sync, relay, or store data when the user goes offline. We'll use those features to create a Database, connect it to the Hub, and connect two peers.
```